youtube_dl_gui/locale/he-IL/LC_MESSAGES/youtube_dl_gui.po

```diff
@@ -0,0 +1,588 @@
+
+#: youtube_dl_gui/formats.py:9 youtube_dl_gui/formats.py:108
+msgid "ID"
+msgstr "מזהה"
+
+#: youtube_dl_gui/formats.py:10 youtube_dl_gui/formats.py:109
+#: youtube_dl_gui/mainframe.py:140
+msgid "Title"
+msgstr "כותרת"
+
+#: youtube_dl_gui/formats.py:11 youtube_dl_gui/formats.py:110
+msgid "Title + ID"
+msgstr "כותרת + מזהה"
+
+#: youtube_dl_gui/formats.py:12 youtube_dl_gui/formats.py:111
+msgid "Title + Quality"
+msgstr "כותרת + איכות"
+
+#: youtube_dl_gui/formats.py:13 youtube_dl_gui/formats.py:112
+msgid "Title + ID + Quality"
+msgstr "כותרת + מזהה + איכות"
+
+#: youtube_dl_gui/formats.py:14 youtube_dl_gui/formats.py:113
+msgid "Custom"
+msgstr "מותאם אישית"
+
+#: youtube_dl_gui/formats.py:19 youtube_dl_gui/formats.py:118
+#: youtube_dl_gui/mainframe.py:503 youtube_dl_gui/mainframe.py:506
+msgid "default"
+msgstr "ברירת מחדל"
+
+#: youtube_dl_gui/mainframe.py:97
+msgid "Enter URLs below"
+msgstr "הזן את כתובת האתר"
+
+#: youtube_dl_gui/mainframe.py:98
+msgid "Update"
+msgstr "עדכון"
+
+#: youtube_dl_gui/mainframe.py:99 youtube_dl_gui/optionsframe.py:41
+msgid "Options"
+msgstr "אפשרויות"
+
+#: youtube_dl_gui/mainframe.py:100 youtube_dl_gui/optionsframe.py:589
+msgid "Stop"
+msgstr "עצור"
+
+#: youtube_dl_gui/mainframe.py:101
+msgid "Info"
+msgstr "מידע"
+
+#: youtube_dl_gui/mainframe.py:102
+msgid "Welcome"
+msgstr "ברוך הבא"
+
+#: youtube_dl_gui/mainframe.py:103
+msgid "Warning"
+msgstr "אזהרה"
+
+#: youtube_dl_gui/mainframe.py:105
+msgid "Add"
+msgstr "הוסף"
+
+#: youtube_dl_gui/mainframe.py:106
+msgid "Download list"
+msgstr "רשימת הורדות"
+
+#: youtube_dl_gui/mainframe.py:107 youtube_dl_gui/mainframe.py:516
+#: youtube_dl_gui/mainframe.py:534
+msgid "Delete"
+msgstr "מחק"
+
+#: youtube_dl_gui/mainframe.py:108
+msgid "Play"
+msgstr "הפעל"
+
+#: youtube_dl_gui/mainframe.py:109
+msgid "Up"
+msgstr "העלה"
+
+#: youtube_dl_gui/mainframe.py:110
+msgid "Down"
+msgstr "הורד"
+
+#: youtube_dl_gui/mainframe.py:111
+msgid "Reload"
+msgstr "טען מחדש"
+
+#: youtube_dl_gui/mainframe.py:112 youtube_dl_gui/mainframe.py:448
+#: youtube_dl_gui/mainframe.py:649
+msgid "Pause"
+msgstr "הפסק"
+
+#: youtube_dl_gui/mainframe.py:113 youtube_dl_gui/mainframe.py:865
+#: youtube_dl_gui/mainframe.py:866 youtube_dl_gui/optionsframe.py:587
+msgid "Start"
+msgstr "התחל"
+
+#: youtube_dl_gui/mainframe.py:114
+msgid "About"
+msgstr "אודות"
+
+#: youtube_dl_gui/mainframe.py:115
+msgid "View Log"
+msgstr "צפה ביומן רישום"
+
+#: youtube_dl_gui/mainframe.py:117
+msgid "Successfully downloaded {0} URL(s) in {1} day(s) {2} hour(s) {3} minute(s) {4} second(s)"
+msgstr "הוריד בהצלחה את {0} כתובות האתרים תוך {1} ימים {2} שעות {3} דקות {4} שניות"
+
+#: youtube_dl_gui/mainframe.py:119
+msgid "Downloads completed"
+msgstr "ההורדה הושלמה"
+
+#: youtube_dl_gui/mainframe.py:120
+msgid "Total Progress: {0:.1f}% | Queued ({1}) Paused ({2}) Active ({3}) Completed ({4}) Error ({5})"
+msgstr "סה"כ התקדמות: {0:.1f}% | בהמתנה ({1}) בהפסקה ({2}) פעילים ({3}) הושלמו ({4}) בשגיאה ({5})"
```

Comment:
Please if you have to use quotes in your translations use single quotes ( ' ) and not double quotes ( " )

Reply:
change to single qoutes